### ISSUE TITLE
Global map disappears when "Show Future Posts" is set to "Only"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cyberhobo/wordpress-geo-mashup#526]
```
I am using the latest versions of Wordpress and Geo Mashup:

I show my users a global map on http://projectcloud.info/globalmap/. I want to 
only show the locations associated with posts whose dates have not passed which 
is why I enable the show only future posts options. Once that is done, the map 
disappears. 

I do not want to show locations associated with all my posts, just the ones 
that are yet to take place. I always fill in the date for the location in the 
geo mashup area while publishing. 

@Dylan: I don't know what some of us would do without this plugin. Excellent 
work!
```

Original issue reported on code.google.com by `chetantr...@gmail.com` on 24 Nov 2011 at 5:29
- Merged into: #491
